team.go

```diff
@@ -134,6 +136,20 @@ func (api *Client) GetAccessLogsContext(ctx context.Context, params AccessLogPar
 	if err != nil {
 		return nil, nil, err
 	}
+
+	for i, login := range response.Logins {
+		switch vv := login.IP.(type) {
+		case string:
+			response.Logins[i].RealIP = vv
+		default:
+			response.Logins[i].RealIP = "0.0.0.0"
+		}
+	}
+
+	for _, login := range response.Logins {
```

Comment:
Apologies for not spotting this in previous reviews! Was the printng here just for debugging?

Reply:
ahh yes totally my bad; forgot to remove it; apologies for not removing in the first place : )